### ISSUE TITLE
About Roboflow-VL Few-Shot Object Detection Challenge 2025

### ISSUE THREAD

**Issue by zhimengXin** [anishmadan23/foundational_fsod#4]
We would like to participate in the Roboflow-VL Challenge 2025, but we are unclear about the rules for using the dataset in the competition. A subset of the Roboflow-VL dataset will be used in this competition. However, the submitted dataset samples include 20 datasets from Roboflow-VL. Additionally, we are required to access and use the nuImages data to generate predictions related to each competition task. Based on this, we need to clearly understand: What is the training set used in the competition? What is the test set? (Is it the complete 20 datasets provided in Roboflow20VL FSOD, or only a few of them?) If you could provide us with a detailed explanation, we would be immensely grateful.

**Comment by neeharperi** [comment]
Thanks for your interest in our challenge! The competition only includes the 20 datasets in [Roboflow20VL-FSOD](https://universe.roboflow.com/roboflow20vl-fsod/). Importantly, we are **not** using nuImages in this iteration of the challenge. If you click on the "dataset" tab, you will see that the train / val / tests splits are defined for each datasets. Please let me know if you have any additional questions, I'd be happy to help address them

**Comment by zhimengXin** [comment]
Thank you very much for your explanation. We will follow the competition rules for the event going forward. However, we currently cannot access the datasets [here](https://universe.roboflow.com/roboflow20vl-fsod/). Could you please confirm if  new datasets will be uploaded in [Roboflow20VL-FSOD](https://universe.roboflow.com/roboflow20vl-fsod/)?

**Comment by neeharperi** [comment]
Thanks for flagging this! I'll let you know once this is resolved

**Comment by zhimengXin** [comment]
Thank you very much for your replies！

**Comment by neeharperi** [comment]
Hi @zhimengXin, the Roboflow20-VL link should be working now. Thanks for your patience!

**Comment by zhimengXin** [comment]
OK Thank you！
